packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -7,6 +7,7 @@
       <div
         v-if="chartTitle"
         class="chart-title"
+        :title="chartTitle"
```

Comment:
this is for accessibility reasons I assume 

@adamdehaven | @portikM - preference on `title` vs `aria-label` here?

Reply:
If you need to hover and see the text, `title`

Otherwise, `aria-label`

Reply:
You may also want a `role="figure` https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/figure_role